### ISSUE TITLE
[Bug]: protobufjs@npm:6.11.4: The remote archive doesn't match the expected checksum

### ISSUE THREAD

**Issue by lly835** [MetaMask/metamask-extension#20638]
### Describe the bug

yarn install 
https://codeload.github.com/MetaMask/metamask-extension/zip/refs/tags/v10.35.0

### Steps to reproduce

1.https://codeload.github.com/MetaMask/metamask-extension/zip/refs/tags/v10.35.0
yarn install

### Version

10.35.0

### Build type

None

### Browser

Chrome

### Operating system

MacOS

**Comment by lly835** [comment]
```
➤ YN0060: │ metamask-crx@workspace:. provides webpack (p9c948) with version 5.82.1, which doesn't satisfy what css-loader requests
➤ YN0002: │ remote-redux-devtools@npm:0.5.16 doesn't provide redux (pb06f7), requested by redux-devtools-instrument
➤ YN0013: │ yocto-queue@npm:0.1.0 can't be found in the cache and will be fetched from the remote registry
➤ YN0013: │ zstd-codec@npm:0.1.4 can't be found in the cache and will be fetched from the remote registry
➤ YN0013: │ zwitch@npm:2.0.4 can't be found in the cache and will be fetched from the remote registry
➤ YN0013: │ zxcvbn@npm:4.4.2 can't be found in the cache and will be fetched from the remote registry
➤ YN0013: │ zxcvbn@patch:zxcvbn@npm%3A4.4.2#./.yarn/patches/zxcvbn-npm-4.4.2-6527983856.patch::version=4.4.2&hash=5e3c67&locator=metamask-crx%40workspace%3A. can't be found in the cache and will be fetch
➤ YN0018: │ protobufjs@npm:6.11.4: The remote archive doesn't match the expected checksum
➤ YN0000: └ Completed in 8s 811ms
➤ YN0000: Failed with errors in 9s 659ms

```

**Comment by Gudahtt** [comment]
v10.34.5 and v10.35.0 both have the wrong checksum for this package. It has been fixed in v10.35.1. If you want to build the older releases, try updating the checksum to match the one in v10.35.1 or installing with `YARN_CHECKSUM_BEHAVIOR=ignore yarn`
